docs/guidance/identifiers.md

```diff
@@ -0,0 +1,38 @@
+# Project identifiers
+
+A project identifier is a globally unique identifier for an infrastructure project. Every project in OC4IDS has a project identifier in the `id` field.
+
+Project identifiers can be used to join up information published at different times or from different systems, for example, including a project identifier in contracting data enables information on the design, construction and supervision contracts for a single infrastructure project to be joined up.
+
+## Project identifiers in contracting data
+
+With an authoritative list of infrastructure projects in a country to draw on, and if the identifiers it assigned are used within contracting systems, it would be possible to automatically check which projects have published contracting information, and to analyse that data to identify projects where further scrutiny, monitoring and data collection should take place.
```

Comment:
> A common need is to access data about the contracting processes related to an infrastructure project. When contracting systems use consistent identifiers to refer to infrastructure projects, this becomes possible. An example use case is automatically checking which projects have related contracting data, and then manually filtering projects for further scrutiny, monitoring or data collection.

Avoid mixing "with…" and "if…" clauses and unmatched verb tenses. Use 'data' instead of 'information'. Use American English for 'analyze'. State the two examples as examples.

This sentence was trying to do a lot. Using three sentences makes it easier to parse. The key concept is consistent identifiers; authoritative lists are one means to achieve that.

Reply:
fixed in https://github.com/open-contracting/infrastructure/pull/97/commits/6dbddcc20150fb09e92a8ab533ffde539d1f8ae7